frontend/packages/console-app/src/components/detect-perspective/PerspectiveConfiguration.tsx

```diff
@@ -151,28 +152,41 @@ const PerspectiveVisibilitySelect: React.FC<{
 
   const [isOpen, setIsOpen] = React.useState(false);
   const selection = options.find((option) => option.isSelected)?.value;
+  const selected = options.find((option) => option.isSelected)?.title;
 
   return (
     <>
-      <SelectDeprecated
-        toggleId={toggleId}
-        disabled={disabled}
-        isOpen={isOpen}
-        selections={selection}
-        onToggle={(_event, isExpanded) => setIsOpen(isExpanded)}
-        onSelect={() => setIsOpen(false)}
-      >
-        {options.map((option) => (
-          <SelectOptionDeprecated
-            key={option.value}
-            value={option.value}
-            description={option.description}
-            onClick={() => onChange(option)}
-          >
-            {option.title}
-          </SelectOptionDeprecated>
-        ))}
-      </SelectDeprecated>
+      <div className="red">
```

Comment:
What is this for?  Leftover from development?

Reply:
Opps... it was an eod push to get it open.